styles.css

```diff
@@ -159,23 +159,24 @@ a[href*="google"]:link {
 
 .data{
   background-color: #FFFFFF;
-  width:22vw;
-  height: 53vh;
+  width:24em;
+  height: 45em;
   text-align: left;
   padding:1.2em 1.1em;
+  display:block;
   position: relative;
   box-shadow: rgba(0, 0, 0, 0.05) 0px 10px 24px 0px, rgba(0, 0, 0, 0.05) 0px 20px 48px 0px, rgba(0, 0, 0, 0.1) 0px 1px 4px 0px;
   border-radius: 0.4em;
 }
 
 #name, #email, #pw {
+  width: 21em;
+  padding:0.5em 0.5em;
   margin-top: 0.3em;
   border: 0.2em solid rgb(236, 236, 236);
```

Comment:
I don't think you really want border width to change, try using `px` here instead